packages/compiler/formatter/print/printer.ts

```diff
@@ -310,12 +311,21 @@ export function printNode(
       return path.call(print, "target");
     case SyntaxKind.Return:
       return printReturnExpression(path as AstPath<ReturnExpressionNode>, options, print);
+    case SyntaxKind.Doc:
+    case SyntaxKind.DocText:
+    case SyntaxKind.DocParamTag:
+    case SyntaxKind.DocTemplateTag:
+    case SyntaxKind.DocReturnsTag:
+    case SyntaxKind.DocUnknownTag:
+      compilerAssert(
```

Comment:
Feel like it could be nice to add formatting for this too in the future.

Reply:
Yes, I'll adjust the comment here and file and link an issue for the future.

Reply:
Filed #1319